build.sbt

```diff
@@ -82,7 +82,8 @@ lazy val core = Project(
   },
   libraryDependencies ++= Seq(
     "org.scala-lang" % "scala-compiler" % scalaVersion.value,
-    "org.scalatest" %% "scalatest" % "3.0.0" % "test"
+    "org.scalatest" %% "scalatest" % "3.0.0" % Test,
+    "com.github.julien-truffaut" %% "monocle-macro" % "1.4.0" % Test
```

Comment:
Instead of relying on an arbitrary project that uses macros, can you just add a test macro?